assets/scripts/modaldialog.js

```diff
@@ -118,7 +118,7 @@ function displayUploadedFiles(jsonstring, filecount, fieldname, show_title, show
         for (i = 0; i < filecount; i++)
         {
             if (isValueInArray(image_extensions, jsonobj[i].ext))
-                display += '<tr><td class="upload image"><img src="' + uploadurl + '/filegetcontents/'+decodeURIComponent(jsonobj[i].filename)+'" class="uploaded" /></td>';
+                display += '<tr><td class="upload image"><img src="' + uploadurl + '/filegetcontents/'+decodeURIComponent(jsonobj[i].filename+'/fieldname/'+fieldname)+'" class="uploaded" /></td>';
```

Comment:
This url need to be tested with GET and PATH url

Reply:
Doesn't it work right now on both? what would be different now?
Still, yes, we could test on both

Reply:
GET URL index.php?r=uploader/index/mode/upload/filegetcontents/fu_ickrap62w67ze4f/fieldname/136931X1X2

We need to review when we rewrite this controller :)